### PR TITLE
[MNOE-103] Change logout button

### PR DESCRIPTION
**Pull request by hedudelgado**
Change logout button to be inside Company select box(top right).
@alexnoox @x4d3

### CODE REVIEW

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 1]
```diff
@@ -21,12 +21,19 @@
...
       </div>
       <div class="admin-link" ng-if="selectBox.user.admin_role">
         <a href="/admin/"><small>{{ 'mno_enterprise.templates.dashboard.menu.access_admin_panel' | translate }}</small></a>
       </div>
+      <hr>
+      <div class="create-new">
+      <a ui-sref="logout" >
+        <i class="fa fa-lock"></i>
+        <small>{{ 'mno_enterprise.templates.dashboard.menu.logout' | translate }}</small></a>
+      </div>
```
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: This looks good.
It just needs to be indented :simple_smile:

File: src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html [hunk 2]
```diff
@@ -21,12 +21,19 @@
...
       </ul>
+      <hr>
       <div class="create-new">
         <a href="" ng-click="selectBox.createNewOrga()"><small>{{ 'mno_enterprise.templates.dashboard.menu.new_company' | translate }}</small></a>
       </div>
       <div class="admin-link" ng-if="selectBox.user.admin_role">
         <a href="/admin/"><small>{{ 'mno_enterprise.templates.dashboard.menu.access_admin_panel' | translate }}</small></a>
       </div>
+      <hr>
+      <div class="create-new">
```
Comment (alexnoox) [inline src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.html]: Can you rename the class `create-new` to `default-link`? Also in the less file.

**Comment by hedudelgado** [comment]
@alexnoox  PR updated

**Comment by alexnoox** [comment]
@hedudelgado: All good 👍

**Pull request merged**